services/gemma-local/transcribe.py

```
@@ -0,0 +1,90 @@
+    
+    # Start server
+    print("Starting Gemma service...")
+    server = subprocess.Popen(
+        [sys.executable, 'main.py'],
+        env={'PORT': '11350'},
+        stdout=subprocess.DEVNULL,
+        stderr=subprocess.DEVNULL
+    )
+    
+    # Wait for server
+    for i in range(30):
```

Ruff (0.12.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)